### DISCUSSION TITLE
配合MSI Afterburner使用时会出现2层监控的情况

### DISCUSSION THREAD

**Discussion by suzushiikazema** [Blinue/Magpie#651] [Q&A] [answered]
![@NAK9M65E9~Q54VT$9B{057](https://github.com/Blinue/Magpie/assets/53015311/1fdab0da-60ce-4c2e-bc6a-335e6b8d0b10)
![3XF(2_~K~`UAB}YK4U$31SW](https://github.com/Blinue/Magpie/assets/53015311/e1e55ea8-faf2-4eac-be14-771e8b4ffe7d)
可能是magpie连带监测本身也一并做放大处理了？是否有办法隐藏或者消去其中一层呢？

**Comment by Blinue** [comment, marked as answer]
https://github.com/Blinue/Magpie/wiki/FAQ#双重监控
